### ISSUE TITLE
Brain2Word: Decoding Brain Activity for Language Generation

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#113]
# TL;DR
A study to understand the letters seen from an fMRI image. fMRI images were used as input to generate GloVE embedding vectors of the five leading candidates from 180 candidate words. Then they do the same thing in GPT-2 with 50257 words and compare the two to solve as a language model. The accuracy is greatly improved compared to the existing methods.
![65730x1](https://user-images.githubusercontent.com/24524018/93408886-9a125480-f8d0-11ea-9593-c6aadfd614ce.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2009.04765

# Submission Dates(yyyy/mm/dd)
2020/09/10

# Authors and institutions
Nicolas Affolter, Beni Egressy, Damian Pascual, Roger Wattenhofer
- ETH Zurich

# Methods

# Results

# Comments
